app/views/dashboards/show.html.erb

```diff
@@ -1,4 +1,4 @@
-<h1 class="t-display page__heading">Dashboard</h1>
+<% @title = t('.title') %>
```

Comment:
@yhirano55 Are we sure that the page title is always the same as the heading on the page? What is the reason we are making the heading a link in this case? You're already on the page right?